packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -208,121 +210,152 @@ export const cancelSignTx = () => (dispatch: Dispatch, getState: GetState) => {
-    const areSatoshisUsed = getAreSatoshisUsed(
-        getState().wallet.settings.bitcoinAmountUnit,
-        account,
-    );
-
-    // get total amount without fee OR token amount
-    const formattedAmount = token
-        ? `${formatAmount(precomposedTx.totalSpent, token.decimals)} ${token.symbol!.toUpperCase()}`
-        : formatNetworkAmount(spentWithoutFee, account.symbol, true, areSatoshisUsed);
-
-    if (sentTx.success) {
-        const { txid } = sentTx.payload;
-        dispatch(
-            notificationsActions.addToast({
-                type: 'tx-sent',
-                formattedAmount,
-                device,
-                descriptor: account.descriptor,
-                symbol: account.symbol,
-                txid,
-            }),
+        const { token } = precomposedTx;
+        const spentWithoutFee = !token
+            ? new BigNumber(precomposedTx.totalSpent).minus(precomposedTx.fee).toString()
+            : '0';
+
+        const areSatoshisUsed = getAreSatoshisUsed(
+            getState().wallet.settings.bitcoinAmountUnit,
+            account,
         );
 
-        if (precomposedTx.prevTxid) {
-            // notification from the backend may be delayed.
-            // modify affected transaction(s) in the reducer until the real account update occurs.
-            // this will update transaction details (like time, fee etc.)
-            dispatch(replaceTransactionThunk({ tx: precomposedTx, newTxid: txid }));
-        }
+        // get total amount without fee OR token amount
+        const formattedAmount = token
+            ? `${formatAmount(
+                  precomposedTx.totalSpent,
+                  token.decimals,
+              )} ${token.symbol!.toUpperCase()}`
+            : formatNetworkAmount(spentWithoutFee, account.symbol, true, areSatoshisUsed);
 
-        // notification from the backend may be delayed.
-        // modify affected account balance.
-        // TODO: make it work with ETH accounts
-        const pendingAccount = getPendingAccount(account, precomposedTx, txid);
-        if (pendingAccount) {
-            // update account
-            dispatch(accountsActions.updateAccount(pendingAccount));
+        if (sentTx.success) {
+            const { txid } = sentTx.payload;
+            dispatch(
+                notificationsActions.addToast({
+                    type: 'tx-sent',
+                    formattedAmount,
+                    device,
+                    descriptor: account.descriptor,
+                    symbol: account.symbol,
+                    txid,
+                }),
+            );
+
+            if (precomposedTx.prevTxid) {
+                // notification from the backend may be delayed.
+                // modify affected transaction(s) in the reducer until the real account update occurs.
+                // this will update transaction details (like time, fee etc.)
+                dispatch(replaceTransactionThunk({ tx: precomposedTx, newTxid: txid }));
```

Comment:
not neccary, txid is available

Reply:
[e683c20](https://github.com/trezor/trezor-suite/pull/7783/commits/e683c20740e77d8e3dc5efd79dbb177aeaabb74e)